docs/user/library/render/map.rst

```diff
@@ -78,7 +78,7 @@ By default, MapViewport does not make any correction for this (this behaviour ac
 WMS specification which states that a map server should honour the bounds provided by the user
 regardless of any distortion which results).
 
-In many cases, it is preferable to have MapViewport automagically correct for differing aspect
+In many cases, it is preferable to have MapViewport automatically correct for differing aspect
```

Comment:
+1 one for changing this to "automatically" because straightforward international English makes documentation more accessible to non-native speakers. While the original text "automagically" was almost certainly intentional, it is jargon and a software community joke and may lead to confusion.